.eslintignore

```diff
@@ -1,3 +1,3 @@
 generators/**/templates
-travis
-docs
+test-integration/samples/app-sample-dev
```

Comment:
Why just ignore this specific directory? I would just ignore the whole directory `test-integration`

Reply:
sure